### PR TITLE
Modify OSEv3.yml by combine(), support optional image upload.

### PR DESCRIPTION
**Pull request by jmencak**
This is a "release early" PR, which has only been partially tested.  @mbruzek PTAL and comment, I'll try to address your comments before merging.  Please do not merge, I'll adjust and merge once fully tested. Thanks.

### CODE REVIEW

File: roles/openshift_on_openstack/tasks/main.yml [hunk 1]
```diff
@@ -158,21 +158,36 @@
...
+  command: "cat {{ osev3_yml_path }}"
+  register: l_osev3_yml
+  changed_when: false
+
+- name: Reading the contents of {{ osev3_yml_cfg_path }}
+  command: "cat {{ osev3_yml_cfg_path }}"
+  register: l_osev3_yml_cfg
+  changed_when: false
+
+- name: Setting facts from {{ osev3_yml_path }} and 'templates/OSEv3.yml.cfg.j2'
```
Comment (mbruzek) [inline roles/openshift_on_openstack/tasks/main.yml]: Use the variable so we can track down the file: "Setting facts from {{ osev3_yml_path }} and {{ osev3_yml_cfg_path}}"

File: vars/openstack.yml [hunk 2]
```diff
@@ -25,3 +25,7 @@ openstack_server_name: "{{ lookup('env', 'server_name')|default('ansible-host',
 openstack_subnet_name: "{{ lookup('env', 'subnet_name')|default('ci_subnet', true) }}"
 # The subnet range to use on the OpenStack private subnet (\23 == 510 addresses).
 openstack_subnet_range: "{{ lookup('env', 'subnet_range')|default('192.168.4.0/23', true) }}"
+# Create flavors in the openstack environment.
+openstack_create_flavors: "{{ lookup('env', 'create_flavors')|default('true', true) }}"
```
Comment (mbruzek) [inline vars/openstack.yml]: I think you can provide true without quotes here.

**Comment by mbruzek** [comment]
I really like the new way of processing the yaml configuration values. Very well done! Also fine with moving the upload images to a new role. This code looks good and has my full support.

**Comment by jmencak** [comment]
Added support for OPENSHIFT_NODE_TARGET during installation, not just scaleup. Had a successful install with v3.9 with this, merging.

**Pull request merged**